andela/hc-rogue-one
# Ft 156577855 team access awareness

#### What does this PR do?
This PR ensures that users get to know that they can access an expanded dashboard that allows teams to collaboratively create and manage alerts based on the landing page UI/UX rework
#### Description of Task to be completed?
When a user wants to know what HealthChecks offers on accessing the landing page, they should be aware they should be aware that they can create teams and access/manage checks as a team.
#### How should this be manually tested?
1. Clone the application and install the requirements in the requirements.txt
2.Run the application and access the landing page
3: There should be a section that informs the user of the features that health checks has to offer.
#### What are the relevant pivotal tracker stories?
#156577855

## Checklist: I completed these to help reviewers




  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`

  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`
